packages/cascara/src/ui/Table/Table.test.js

```diff
@@ -25,69 +24,63 @@ describe('Table', () => {
   // The test `row actions` corresponds to condition a, whilst the test
   // `editable records` addresses condition b.
   describe('component tree', () => {
-    const whileTheUIisReady = async (miliseconds) =>
-      await new Promise((resolve) => setTimeout(resolve, miliseconds));
     const datasetSize = 100;
```

Comment:
@lordpixel I was trying to take a look at the differences in the snapshot file... and Github cannot load the diff. The snapshot itself is over 30,000 lines! ha

I think this data set is far too large to render the table. I am pretty sure this test in particular runs VERY slow. Also, a data set of 100 records is not how we are using the table as of today. Let's think about a few things here:

- What is this test supposed to help us catch?
- If the test fails, is it going to be easy to tell what went wrong and where?
- Does the test evaluate a real use case?

I think for the first two questions, we do answer that in the comments about the test. Basically this is kind of a smoke test for he table to make sure it will correctly render data based on our API. The Snapshot should tell us if there were markup changes. That makes sense. The name component tree I think also makes sense because this is to make sure it was consistent.

Really this comes down to the last question then. I think 100 records is probably way too large here for this test. Will the test be just as effective rendering 5, 10, 15 records? Probably. I think it will also run much faster and if there are changes in the snapshot, we won't have to look at a diff with 30,000 lines. :)

Reply:
@brianespinosa you are right, in fact, when I work on unit tests for Table I always set the `datasetSize` to 5. I didn't know the snapshots were so large, this definitely has to change! Let me work on it, thanks for the catch!

Reply:
This has been changed to 10, snapshots are smaller now.